exercicios/para-casa/entregas/erika-pimentel/projeto/ContaStandard/ContaStandard.test.js

```diff
@@ -0,0 +1,149 @@
+const ContaStandard = require("./ContaStandard");
+
+describe("Testes da Classe ContaStandard", () => {
+    test("verificar se instancia foi criada corretamente", () => {
+      const contaStandard = new ContaStandard();
+      expect(contaStandard instanceof ContaStandard).toBe(true);
+      
+      contaStandard.destruir()
+  });
+
+
+  test("instanciar conta standard com dados válidos e renda compatível", () => {
```

Comment:
Parabéns pelos testes. Ficaram muito claros. 🎊🎊 Muito bem estruturados. Deixo como sugestão para uma próxima aplicar a prática de começar os nomes de teste com  "deve" (ou "should", em inglês) que é uma convenção popular na metodologia de desenvolvimento  Behavior-Driven Development (BDD).  É apenas uma convenção e não é uma regra estrita, mas uma prática  bem recomendada.